web/src/app/admin/AdminAPIKeys.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from 'react'
+import makeStyles from '@mui/styles/makeStyles'
+import {
+  Button,
+  Grid,
+  Typography,
+  Card,
+  ButtonBase,
+  CardHeader,
+} from '@mui/material'
+import { Add } from '@mui/icons-material'
+import AdminAPIKeysDrawer from './admin-api-keys/AdminAPIKeysDrawer'
+import { GQLAPIKey, CreatedGQLAPIKey } from '../../schema'
+import { Time } from '../util/Time'
+import { gql, useQuery } from '@apollo/client'
+import FlatList, { FlatListListItem } from '../lists/FlatList'
+import Spinner from '../loading/components/Spinner'
+import { GenericError } from '../error-pages'
+import { Theme } from '@mui/material/styles'
+import AdminAPIKeysActionDialog from './admin-api-keys/AdminAPIKeysActionDialog'
+import AdminAPIKeysTokenDialog from './admin-api-keys/AdminAPIKeysTokenDialog'
+// query for getting existing API Keys
+const getAPIKeysQuery = gql`
+  query gqlAPIKeysQuery {
+    gqlAPIKeys {
+      id
+      name
+      description
+      createdAt
+      createdBy {
+        id
+        role
+        name
+        email
+      }
+      updatedAt
+      updatedBy {
+        id
+        role
+        name
+        email
+      }
+      lastUsed {
+        time
+        ua
+        ip
+      }
+      expiresAt
+      allowedFields
+      role
+    }
+  }
+`
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    '& .MuiListItem-root': {
+      'border-bottom': '1px solid #333333',
+    },
+  },
+  buttons: {
+    'margin-bottom': '15px',
+  },
+  containerDefault: {
+    [theme.breakpoints.up('md')]: {
+      maxWidth: '100%',
+      transition: `max-width ${theme.transitions.duration.leavingScreen}ms ease`,
+    },
+    '& .MuiListItem-root': {
+      padding: '0px',
+    },
+  },
+  containerSelected: {
+    [theme.breakpoints.up('md')]: {
+      maxWidth: '70%',
+      transition: `max-width ${theme.transitions.duration.enteringScreen}ms ease`,
+    },
+    '& .MuiListItem-root': {
```

Comment:
I'm not sure if these classnames are stable/should be referenced directly, @Forfold can you weigh in?

Reply:
I believe the recommended solution is to use the `sx` prop for styling. I removed this locally and didn't see a change, so I think it can be removed